[2024-01-07 19:51:28,206] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T18:00:00+00:00 [queued]>
[2024-01-07 19:51:28,223] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T18:00:00+00:00 [queued]>
[2024-01-07 19:51:28,223] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-07 19:51:28,223] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-01-07 19:51:28,223] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-07 19:51:28,251] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): checking_model_prediction_ratio> on 2024-01-07 18:00:00+00:00
[2024-01-07 19:51:28,256] {standard_task_runner.py:52} INFO - Started process 13569 to run task
[2024-01-07 19:51:28,258] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_inference_pipeline', 'checking_model_prediction_ratio', 'scheduled__2024-01-07T18:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/lead_scoring_inference_pipeline/.ipynb_checkpoints/lead_scoring_inference_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmpqcfqac1b', '--error-file', '/tmp/tmpvsbzj645']
[2024-01-07 19:51:28,259] {standard_task_runner.py:80} INFO - Job 48: Subtask checking_model_prediction_ratio
[2024-01-07 19:51:28,343] {task_command.py:371} INFO - Running <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T18:00:00+00:00 [running]> on host a3830b6d5f93
[2024-01-07 19:51:28,544] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_scoring_inference_pipeline
AIRFLOW_CTX_TASK_ID=checking_model_prediction_ratio
AIRFLOW_CTX_EXECUTION_DATE=2024-01-07T18:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-07T18:00:00+00:00
[2024-01-07 19:51:28,550] {logging_mixin.py:115} INFO - Error while running prediction_ratio_check : Execution failed on sql 'select * from predictions': no such table: predictions
[2024-01-07 19:51:28,550] {python.py:173} INFO - Done. Returned value was: None
[2024-01-07 19:51:28,565] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_inference_pipeline, task_id=checking_model_prediction_ratio, execution_date=20240107T180000, start_date=20240107T195128, end_date=20240107T195128
[2024-01-07 19:51:28,631] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-07 19:51:28,696] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
